tools/src/linter/InlineObjectSchemaValidator.ts

```diff
@@ -10,8 +10,8 @@
 import type NamespacesFolder from './components/NamespacesFolder'
 import type SchemasFolder from './components/SchemasFolder'
 import { type ValidationError } from 'types'
-import { SchemaVisitor } from './utils/SpecificationVisitor'
-import { is_ref, type MaybeRef, SpecificationContext } from './utils'
+import { SchemaVisitor } from '../_utils/SpecificationVisitor'
+import { is_ref, type MaybeRef, SpecificationContext } from '../_utils'
```

Comment:
What's the reason for the leading underscore on `_utils`?

Reply:
To differentiate it from other folders like `tester` `linter` `merger` where they are tools while `_utils` is not a tool.